components/core/tools/scripts/lib_install/pipx-packages/install-pipx.sh

```
@@ -0,0 +1,59 @@
+    ${pipx_upgrade_cmd} >/dev/null 2>&1 || true
+fi
+
+# Find pipx path and validate version
+pipx_bin="$(command -v pipx 2>/dev/null || true)"
+if [ -z "${pipx_bin:-}" ]; then
+    echo "Error: failed to automatically install pipx. Please install it manually."
+    exit 1
+fi
+
+installed_version="$("${pipx_bin}" --version)"
+IFS=. read -r installd_version_major installed_version_minor _ <<<"${installed_version}"
```

Shellcheck (0.11.0)

[warning] installd_version_major appears unused. Verify use (or export if used externally).

(SC2034)